go.mod

```diff
@@ -5,8 +5,10 @@ go 1.22.6
 require (
 	github.com/aws/aws-sdk-go-v2/credentials v1.17.47
 	github.com/aws/smithy-go v1.22.1
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
```

Comment:
The github says it's deprecated in favor of go-grpc-middleware

Reply:
Maybe you can check if you can prefix those metrics with `s3_cosi_` so we can easily identify them

Reply:
This is not an S3 service so we will prefix it with `cosi_driver` for custom metrics but for default grpc metrics we should keep the convention of grpc_, just like we do for HTTP protocol.

Metrics such as grpc_server_started_total convey that these are standard gRPC server metrics, helping tools or dashboards that expect gRPC naming conventions to process them without requiring additional configuration.

Reply:
I will check this out go-grpc-middleware

Reply:
For custom metrics we can add that
So default can be cosi_driver prefix which is configurable
But I would like to keep grpc metrics in standard format .